ios/Classes/FlutterSecureStoragePlugin.m

```diff
@@ -112,25 +121,26 @@ - (void)write:(NSString *)value forKey:(NSString *)key forGroup:(NSString *)grou
         NSDictionary *update = @{
             (__bridge id)kSecValueData: [value dataUsingEncoding:NSUTF8StringEncoding],
             (__bridge id)kSecAttrAccessible: (__bridge id) attrAccessible,
+            (__bridge id)kSecAttrSynchronizable: (__bridge id) attrSynchronizable,
         };
         
         status = SecItemUpdate((__bridge CFDictionaryRef)search, (__bridge CFDictionaryRef)update);
-        if (status != noErr){
             NSLog(@"SecItemUpdate status = %d", (int) status);
+        if (status != noErr){
```

Comment:
Why is the NSLog moved outside the `noErr` block?

Reply:
Fixed